bot.py

```diff
@@ -5,18 +5,11 @@
 
 
 from mapgen import *  # Did I set up the module correctly?
+from dice import *
 from secrets import token
 from random import randint
 from PIL import Image  # ! DEPENDENCY ! Pillow module is not standard. Also, if I import this as part of another module, then import THAT module, then I *shouldn't* need to import it again, correct?
```

Comment:
> \# ! DEPENDENCY ! Pillow module is not standard.

Not a problem. We already have other external dependencies, which you can see listed in `requirements.txt` and `setup.py`. 😃 

We can talk a bit about adding and managing dependencies in chat.

> Also, if I import this as part of another module, then import THAT module, then I *shouldn't* need to import it again, correct?

If you don't need to access it directly here, then no, there's no need to import it.

And if you did want to access it here, you could (but *shouldn't*) do something like this:

In `b.py`:
```python
import os
```
In `a.py`:
```python
import b
print(dir(b))
from b import os
print(os)
import os as this_os
print(this_os == os)
```

```bash
$ python a.py
['__builtins__', '__doc__', '__file__', '__name__', '__package__', 'os']
<module 'os' from '/your/path/to/lib/python2.7/os.pyc'>
True
```